viral/imaging_utils.py

```
@@ -4,7 +4,8 @@
 from scipy.ndimage import gaussian_filter1d
 from typing import List, Tuple
 import numpy as np
-from viral.models import TrialInfo
+from viral.models import TrialInfo, WheelFreeze
+from viral.models import TrialInfo, WheelFreeze
```

Ruff (0.8.2)

Redefinition of unused `TrialInfo` from line 7

Remove definition: `TrialInfo`

(F811)